tests/test_qpu_braket.py

```diff
@@ -360,37 +362,56 @@ def test_exceptions_in_init(self):
         aws_device = DeviceAWS('arn:aws:braket:::device/quantum-simulator/amazon/sv1')
         
         QAOAAWSQPUBackend(circuit_params, aws_device, shots, None, None, True, 1.)
-     
+    
+    @pytest.mark.qpu            
+    def test_remote_qubit_overflow(self):
```

Comment:
I dont fully understand this test? How would it overflow? SV1 should be able to run 10 qubits, no?

Reply:
Oh I see, you are basically creating a 100 qubit cost Hamiltonian, and expecting that for SV1, AWS will output an overflow error? Did I understand it correctly?